upf/pfcp.c

```diff
@@ -5395,6 +5404,10 @@ static struct pfcp_group_ie_def pfcp_update_urr_group[] =
       .type = PFCP_IE_QUOTA_HOLDING_TIME,
       .offset = offsetof(pfcp_update_urr_t, quota_holding_time)
     },
+    [UPDATE_URR_QUOTA_VALIDITY_TIME] = {
+      .type = PFCP_IE_QUOTA_VALIDITY_TIME,
+      .offset = offsetof(pfcp_update_urr_t, quota_validity_time)
+    },
```

Comment:
wrong order, Quota Validity Time should go after Dropped DL Traffic Threshold to keep it in sync with TS 29.244